R/getStatistics.R

```diff
@@ -53,7 +53,7 @@
 #' @param includeAnnotations Includes FCS file annotations in the returned data
```

Comment:
Can you add some text like "specify neither \code{fcsFileIds} nor \{fcsFiles} to calculate statistics for all non-control FCS files" and likewise for populationIds please?